src/memento_test.jl

```diff
@@ -15,6 +15,7 @@ occursin_msg(s::AbstractString, output) = occursin(s, output)
 occursin_msg(s::Regex, output) = occursin(s, output)
 occursin_msg(s::Function, output) = s(output)
 occursin_msg(S::Union{AbstractArray,Tuple}, output) = all(s -> occursin_msg(s, output), S)
+occursin_msg(s, ::Nothing) = false  # Fallback, but shouldn't be needed
```

Comment:
This comment confuses me

Reply:
It's cause our test logger is always setting both the found level and message only when we actually find something, so if `handler.found[1] == $(esc(level))` returns true then `occursin_msg($(esc(msg)), handler.found[2])` will not hit this dispatch. The reason we were hitting this before is because is because we had separate `@test` calls for those two conditions, now we just have 1 short-circuited test.

Reply:
So this is dead code.
Lets not have it then?

Reply:
👌